myaws/ssm_parameter_env.go

```diff
@@ -15,21 +15,29 @@ type SSMParameterEnvOptions struct {
 
 // SSMParameterEnv prints SSM parameters as a list of environment variables.
 func (client *Client) SSMParameterEnv(options SSMParameterEnvOptions) error {
-	metadata, err := client.FindSSMParameterMetadata(options.Name)
-	if err != nil {
-		return err
-	}
+	var parameters []*ssm.Parameter
+	if strings.HasPrefix(options.Name, "/") {
```

Comment:
For GetParametersByPath, a path is required parameter and must start with `/`, so this check makes sense not to break backward compatibility, but we should add more context comments why we need the GetParametersByPath API. I think it's not obvious without knowing the rate limit issue.

Reply:
I added comment.